emails/shared/base-email.pug

```diff
@@ -1,40 +1,40 @@
 doctype html
 // all styles/assets need to be relative to the "emails" directory AKA /emails
-link(rel="stylesheet", href="/shared/style.css", data-inline)
+link(rel="stylesheet", href="./style.css", data-inline)
 html
     body
-        - const emailAssetsUrl = `${baseUrl}/emails`
+        - const emailAssetsUrl = '../../public/emails'
```

Comment:
This won't work when the e-mails are actually sent out. It will only work locally on your file system. Keep the `${baseUrl}` because it needs to point to a resolvable link